models/Article.php

```diff
@@ -194,15 +184,16 @@ function delete_article()
         $db = getPdo();
         
         if (empty($_GET['id']) || !ctype_digit($_GET['id'])) {
-            die("Ho ! Fallait préciser le paramètre id en GET !");
+            echo("Ho ! Fallait préciser le paramètre id en GET !");
+        }
+        if(isset($_GET['id'])){
+            $id = $_GET['id'];
         }
-
-        $id = $_GET['id'];
 
         $query = $db->prepare('SELECT * FROM articles WHERE id = :id');
         $query->execute(['id' => $id]);
         if ($query->rowCount() === 0) {
-            die("Aucun commentaire n'a l'identifiant $id !");
+            echo("Aucun commentaire n'a l'identifiant $id !");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)